roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_priorities.py

```diff
@@ -70,6 +70,12 @@ def run(self, terms, variables=None, zones_enabled=True, short_version=None,
         if short_version not in ['1.1', '1.2', '1.3']:
             priorities.append({'name': 'InterPodAffinityPriority', 'weight': 1})
 
+        if short_version not in ['1.1', '1.2', '1.3', '1.4']:
+            priorities.append({'name': 'LeastRequestedPriority', 'weight': 1})
+
+        if short_version not in ['1.1', '1.2', '1.3', '1.4']:
+            priorities.append({'name': 'BalancedResourceAllocation', 'weight': 1})
+
```

Comment:
Should the tests for the priorities lookup plugin be updated for these changes as well?